app/src/main/java/nextstep/github/ui/home/RepositoryItemScreen.kt

```diff
@@ -29,28 +36,55 @@ fun RepositoryItem(
             ),
         verticalArrangement = Arrangement.Center
     ) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(
+                text = if (repository.isHot) "HOT" else "",
+                style = MaterialTheme.typography.labelLarge,
+                color = MaterialTheme.colorScheme.primary,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis
+            )
+            TextWithIcon(
+                text = "${repository.star}",
```

Comment:
다음과 같이 작성해도 충분할 것 같아요!
```suggestion
                text = repository.star.toString(),
```

Reply:
string에 대해서 참조를 넣는것처럼 보이지 않으니 훨씬 나아 보이네요